validate/validate.go

```diff
@@ -490,6 +498,14 @@ func namespaceValid(ns rspec.Namespace) bool {
 	return true
 }
 
+func nestedValid(bPath, fPath string) bool {
+	if fPath == bPath {
+		return false
+	}
+
+	return strings.HasPrefix(bPath, fPath)
```

Comment:
`strings.HasPrefix` is not what you want to use (e.g. `strings.HasPrefix("/ab", "/a")` is true, but `/ab` and `/a` are siblings).  What you want is something like [`filepath.Rel`][1], but using the path syntax appropriate for the `config.json`'s declared OS instead of the path syntax for which `oci-runtime-tool` was compiled.  So `windowsfilepath.Rel` or `posixfilepath.Rel`, depending on `platform.os`.

[1]: https://golang.org/pkg/path/filepath/#Rel